wordpress-mobile/WordPress-Android
# Stats Posting activity block

Fixes #9289 - Related FluxC changes - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1135
This PR adds a posting activity block in the Insights. The `View more` screen will be implemented in another PR since it requires the other `View more` screens to be merged first. 
The block contains at most 3 months, stretched horizontally. If the months cannot fit the screen, we'll hide them (we always show at least one month worth of data). The data is not clickable. 
There are 5 color levels visible in the blocks. The light grey is when there shows when the number of posts for the day is 0. The other colors are divided by 25%.
The data might be slightly different than what you see in Calypso because I'm using local Calendar to decide whether an event happened on a certain day. If (for example) there was an event happening in the evening in the US, it would be already happening after midnight of my local time and that makes the posting activity graphs slightly different. We might decide 

To test:
* Go to Stats/Insights
* Scroll to the `Posting activity` block
* Check that you see three months when they fit the screen.
* Check on a smaller device that fewer blocks are shown

<img width="303" alt="screenshot 2019-02-26 at 10 39 47" src="https://user-images.githubusercontent.com/1079756/53404399-e2e47e80-39b5-11e9-9c4c-d2d8dccbdc3a.png">
<img width="242" alt="screenshot 2019-02-26 at 11 00 44" src="https://user-images.githubusercontent.com/1079756/53404400-e2e47e80-39b5-11e9-92d8-32341db75031.png">

Update release notes:
